src/pageNotFound/pageNotFound.component.tsx

```diff
@@ -16,12 +17,12 @@ const styles = (theme: Theme): StyleRules => ({
   bugIcon: {
     width: '10vw',
     height: '10vw',
-    color: theme.palette.primary.main,
+    color: (theme as UKRITheme).ukri.bright.blue,
```

Comment:
Looks good. One small change I would suggest is using ukri.contrast.blue instead of bright since then it would match the other contrast changes. Its the same blue on light mode and a lighter one on dark mode.